### ISSUE TITLE
MissingValueException when running on Jenkins

### ISSUE THREAD

**Issue by ncreep** [codacy/sbt-codacy-coverage#36]
Hi,

I'm trying to run Codacy (the `codacyCoverage` command) on a Jenkins server and  I get the following output:
```
14:04:15 [info] Uploading coverage data...

14:04:18 rapture.data.MissingValueException: missing value: success
14:04:18 	at rapture.data.DataType$$anonfun$doNormalize$1$$anonfun$apply$2.liftedTree2$1(data.scala:240)
14:04:18 	at rapture.data.DataType$$anonfun$doNormalize$1$$anonfun$apply$2.apply(data.scala:232)
14:04:18 	at rapture.data.DataType$$anonfun$doNormalize$1$$anonfun$apply$2.apply(data.scala:227)
14:04:18 	at rapture.core.package$$anonfun$yCombinator$1.apply(package.scala:70)
14:04:18 	at rapture.data.DataType$class.doNormalize(data.scala:251)
14:04:18 	at rapture.json.Json.doNormalize(json.scala:149)
14:04:18 	at rapture.data.DataType$class.$normalize(data.scala:206)
14:04:18 	at rapture.json.Json.$normalize(json.scala:149)
14:04:18 	at rapture.json.Extractors_1$$anon$2$$anonfun$extract$4$$anonfun$apply$2.apply(extractors.scala:64)
14:04:18 	at rapture.json.Extractors_1$$anon$2$$anonfun$extract$4$$anonfun$apply$2.apply(extractors.scala:64)
14:04:18 	at rapture.core.Mode$class.catching(modes.scala:72)
14:04:18 	at rapture.core.ThrowExceptionsMode.catching(modes.scala:198)
14:04:18 	at rapture.json.Extractors_1$$anon$2$$anonfun$extract$4.apply(extractors.scala:64)
14:04:18 	at rapture.json.Extractors_1$$anon$2$$anonfun$extract$4.apply(extractors.scala:64)
14:04:18 	at rapture.core.ThrowExceptionsMode.wrap(modes.scala:200)
14:04:18 	at rapture.json.Extractors_1$$anon$2.extract(extractors.scala:64)
14:04:18 	at rapture.json.Extractors_1$$anon$2.extract(extractors.scala:61)
14:04:18 	at rapture.data.DataType$class.as(data.scala:256)
14:04:18 	at rapture.json.Json.as(json.scala:149)
14:04:18 	at com.codacy.api.service.CoverageServices$$anon$1$$anonfun$extract$1$$anonfun$apply$1.apply(CoverageServices.scala:20)
14:04:18 	at rapture.core.ThrowExceptionsMode.unwrap(modes.scala:201)
14:04:18 	at rapture.core.Mode$class.unwrap(modes.scala:53)
14:04:18 	at rapture.core.ThrowExceptionsMode.unwrap(modes.scala:198)
14:04:18 	at com.codacy.api.service.CoverageServices$$anon$1$$anonfun$extract$1.apply(CoverageServices.scala:20)
14:04:18 	at com.codacy.api.service.CoverageServices$$anon$1$$anonfun$extract$1.apply(CoverageServices.scala:20)
14:04:18 	at rapture.core.ThrowExceptionsMode.wrap(modes.scala:200)
14:04:18 	at com.codacy.api.service.CoverageServices$$anon$1.extract(CoverageServices.scala:20)
14:04:18 	at com.codacy.api.service.CoverageServices$$anon$1.extract(CoverageServices.scala:20)
14:04:18 	at rapture.data.DataType$class.as(data.scala:256)
14:04:18 	at rapture.json.Json.as(json.scala:149)
14:04:18 	at com.codacy.api.client.CodacyClient.post(CodacyClient.scala:60)
14:04:18 	at com.codacy.api.service.CoverageServices.sendReport(CoverageServices.scala:20)
14:04:18 	at com.codacy.CodacyCoveragePlugin$$anonfun$7.apply(CodacyCoveragePlugin.scala:77)
14:04:18 	at com.codacy.CodacyCoveragePlugin$$anonfun$7.apply(CodacyCoveragePlugin.scala:64)
14:04:18 	at com.codacy.api.helpers.FileHelper$$anonfun$withTokenAndCommit$1$$anonfun$apply$2.apply(FileHelper.scala:21)
14:04:18 	at com.codacy.api.helpers.FileHelper$$anonfun$withTokenAndCommit$1$$anonfun$apply$2.apply(FileHelper.scala:19)
14:04:18 	at scala.Option.map(Option.scala:145)
14:04:18 	at com.codacy.api.helpers.FileHelper$$anonfun$withTokenAndCommit$1.apply(FileHelper.scala:19)
14:04:18 	at com.codacy.api.helpers.FileHelper$$anonfun$withTokenAndCommit$1.apply(FileHelper.scala:18)
14:04:18 	at com.codacy.api.helpers.FileHelper$$anonfun$withCommit$2.apply(FileHelper.scala:34)
14:04:18 	at com.codacy.api.helpers.FileHelper$$anonfun$withCommit$2.apply(FileHelper.scala:33)
14:04:18 	at scala.Option.map(Option.scala:145)
14:04:18 	at com.codacy.api.helpers.FileHelper$.withCommit(FileHelper.scala:33)
14:04:18 	at com.codacy.api.helpers.FileHelper$.withTokenAndCommit(FileHelper.scala:18)
14:04:18 	at com.codacy.CodacyCoveragePlugin$.com$codacy$CodacyCoveragePlugin$$codacyCoverageCommand(CodacyCoveragePlugin.scala:64)
14:04:18 	at com.codacy.CodacyCoveragePlugin$AutoImport$$anonfun$baseSettings$2.apply(CodacyCoveragePlugin.scala:31)
14:04:18 	at com.codacy.CodacyCoveragePlugin$AutoImport$$anonfun$baseSettings$2.apply(CodacyCoveragePlugin.scala:30)
14:04:18 	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
14:04:18 	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
14:04:18 	at sbt.std.Transform$$anon$4.work(System.scala:63)
14:04:18 	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
14:04:18 	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
14:04:18 	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
14:04:18 	at sbt.Execute.work(Execute.scala:235)
14:04:18 	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
14:04:18 	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
14:04:18 	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
14:04:18 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
14:04:18 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
14:04:18 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
14:04:18 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
14:04:18 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
14:04:18 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
14:04:18 	at java.lang.Thread.run(Thread.java:745)
14:04:18 [error] (new-delivery/*:codacyCoverage) rapture.data.MissingValueException: missing value: success
```

It seems that [this](https://github.com/codacy/sbt-codacy-coverage/blob/48bf924727ed0f8e72f564e49311c4343257c67d/src/main/scala/com/codacy/CodacyCoveragePlugin.scala#L83) is the line of code responsible for the error. I'm also guessing, though not quite certain, that the coverage report was sent back to the Codacy server, and it's just the logging function trying to call an invalid field.

Is there anything that I can do to work around it? 

Thanks.

**Comment by rtfpessoa** [comment]
@mrfyda can you take a took?

**Comment by mrfyda** [comment]
Fixed in 1.3.7

Thanks for reporting!
